base/rational.jl

```diff
@@ -98,9 +98,26 @@ function //(x::Rational, y::Rational)
 end
 
 //(x::Complex, y::Real) = complex(real(x)//y, imag(x)//y)
-//(x::Number, y::Complex) = x*conj(y)//abs2(y)
-
-
+function //(x::Number, y::Complex)
+    if((x//abs2(y))==0//1 || (x//abs2(y))==1//0)
+        return (x//abs2(y))
+    end
+    return (x//abs2(y))*conj(y)
+end
+function //(x::Number, y::Complex{<:Integer})
+    if isinf(real(y)) || isinf(imag(y))
+        return 0//1
+    end
```

Comment:
If y is a complex integer isn't `isinf` here always false?

Reply:
In issue #56245 the errors were thrown for "incorrectly assumes y is finite and nonzero" so I added conditions for for infinite and 0.

Reply:
Did this get resolved?

Reply:
Yes! I have changed the implementation now. This was regarding an outdated issue. mb for not resolving.